src/ResourceManager/Profile/Commands.Profile/Context/GetAzureRMContext.cs

```diff
@@ -26,6 +27,26 @@ namespace Microsoft.Azure.Commands.Profile
     [OutputType(typeof(PSAzureContext))]
     public class GetAzureRMContextCommand : AzureRMCmdlet
     {
+        /// <summary>
+        /// Gets the current default context.
+        /// </summary>
+        protected override AzureContext DefaultContext
```

Comment:
@shahabhijeet it's found in [`System.Management.Automation`](https://msdn.microsoft.com/en-us/library/system.management.automation.errorcategory(v=vs.85).aspx)

I will add a test as well